spec/lib/opentoken-lib.spec.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+describe("opentokenLib", () => {
+    var opentokenLib, req, signed;
+
+    // Just some test data that is constant.
+    const account = {
+            code: "code",
+            secret: "secret",
+            id: "accountId"
+        },
+        host = "api.opentoken.io";
+
+    beforeEach(() => {
+        opentokenLib = require("../../lib/opentoken-lib")(host, "info");
+        signed = opentokenLib.createSignedTokenRequests(account.id, account.code, account.secret);
+        req = opentokenLib.createTokenRequest(account.id);
```

Comment:
Not exactly sure what you are saying. I did it this way because of the possibility of adding multiple tests. I want to create this factory in the beforeEach. Perhaps I am not understanding